src/components/event_merchant/login.vue

```diff
@@ -13,7 +13,7 @@
                             <div class="error_message" v-if="errors">
                                 <p v-for="error in errors" :key="error">{{error}}</p>
                             </div>
-                            <form  @submit.prevent="submit"><!-- -->
+                            <form  @submit.prevent="getSubmit"><!-- -->
```

Comment:
Can you rename the `getsubmit` method be more descriptive?

Reply:
This has been resolved.